<launch>
    <param name="robot_description" command="cat $(find omni_description)/urdf/omni.urdf" />
	<arg name="reference_frame" default="/map"/>
	<arg name="units" default="mm"/>
	<arg name="publish_rate" default="1000"/>
	<arg name="omni_name" default="phantom"/>
 
    <node name="omni_state" pkg="omni_common" type="omni_state" output="screen"> 
	<param name="omni_name" value="$(arg omni_name)"/>
	<param name="publish_rate" value="$(arg publish_rate)"/>
	<param name="reference_frame" value="$(arg reference_frame)"/>
	<param name="units" value="$(arg units)"/>
	</node>
 
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
	</node>
	
    <!-- <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
	</node> -->
	<node name="robot_state_publisher_omni" pkg="robot_state_publisher" type="robot_state_publisher">
	<remap from="/joint_states" to="/joint_states_omni" />
	</node>
 
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find omni_common)/launch/omni.rviz" required="true" />
 
</launch>
